sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/ExecutorResourceMapper.java

```diff
@@ -32,7 +31,6 @@ public ExecutorResourceMapper(
             Collection<ResourceSpec> resourceSpecs,
             Protos.ExecutorInfo executorInfo,
             boolean useDefaultExecutor) {
-        this.executorInfo = executorInfo;
         this.volumeSpecs = podSpec.getVolumes();
```

Comment:
Is the member unused? Would it make sense to create a follow-up ticket to change the constructor argument list? I assume that it makes sense since `ExecutorResourceMapper` doesn't implement or extend anything.

Reply:
The `executorInfo`'s resources are extracted a couple lines down from here. It's just the `executorInfo` member itself that was unused.

Reply:
Me and my lazyness in expanding github diffs.